src/cli/index.ts

```diff
@@ -143,29 +146,32 @@ export default class Cli {
           )
           type = 'progress'
         }
-        return FormatterBuilder.build(type, typeOptions)
+        return await FormatterBuilder.build(type, typeOptions)
       })
     )
     return async function () {
       await Promise.all(formatters.map(async (f) => await f.finished()))
     }
   }
 
-  getSupportCodeLibrary({
+  async getSupportCodeLibrary({
     newId,
     supportCodeRequiredModules,
     supportCodePaths,
-  }: IGetSupportCodeLibraryRequest): ISupportCodeLibrary {
+  }: IGetSupportCodeLibraryRequest): Promise<ISupportCodeLibrary> {
     supportCodeRequiredModules.map((module) => require(module))
     supportCodeLibraryBuilder.reset(this.cwd, newId)
-    supportCodePaths.forEach((codePath) => {
-      try {
+    for (const codePath of supportCodePaths) {
+      if (
+        codePath.endsWith('.js') ||
+        codePath.endsWith('.mjs') ||
+        codePath.endsWith('.cjs')
+      ) {
+        await importer(pathToFileURL(codePath))
+      } else {
         require(codePath)
```

Comment:
We've lost the `try ... catch` around `require(codePath)` in the process. Is it on purpose?

Reply:
No - thanks, good catch. Re-added.